README.md

```diff
@@ -14,7 +14,7 @@ steps:
     plugins:
       - automattic/git-s3-cache#v1.0.0:
           bucket: "my-s3-bucket"
-          repo: "path-to-repo-in-s3/" # note the trailing slash – it must be present
+          repo: "path-to-repo-in-s3/"
```

Comment:
> That comment is not true anymore, now that https://github.com/Automattic/git-s3-cache-buildkite-plugin/pull/6 landed.

Should we remove the trailing slash from the recommendation, then, to keep things simpler?

```suggestion
          repo: "path-to-repo-in-s3"
```

Reply:
We could … but auto-merge 😅
We could also switch the example to something like `org/project` since that's the recommended dollar structure to use in S3 in the rest of the README 🙃